client/extensions/woocommerce/app/product-categories/update.js

```diff
@@ -76,20 +88,73 @@ class ProductCategoryUpdate extends React.Component {
 		}
 	}
 
-	onTrash = () => {};
+	onDelete = () => {
+		const { translate, site, category, deleteProductCategory: dispatchDelete } = this.props;
+		const areYouSure = translate( "Are you sure you want to permanently delete '%(name)s'?", {
+			args: { name: category.name },
+		} );
+		accept( areYouSure, function( accepted ) {
+			if ( ! accepted ) {
+				return;
+			}
+			const successAction = () => {
+				debounce( () => {
+					page.redirect( getLink( '/store/products/categories/:site/', site ) );
+				}, 2000 )();
+				return successNotice(
+					translate( '%(name)s successfully deleted.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **76** times:
`translate( '%s was successfully deleted.', { context: 'theme'} )` *ES Score: 7*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-product-cat-save)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).